config/gpu_configs/target_gpu_implicit_baroclinic_wave.yml

```diff
@@ -0,0 +1,7 @@
+h_elem: 30
+initial_condition: "DryBaroclinicWave"
+t_end: "1days"
+z_elem: 45
+dt: "50secs"
```

Comment:
do we know for sure that dt=50secs would not lead the simulation to blow up? (Perhaps write the corresponding CFL condition numbers in a comment?)

Reply:
We should verify this using a long run!